recipe/meta.yaml

```diff
@@ -25,25 +25,31 @@ requirements:
     - python >=3.6
     # Required
     - affine >=2.2
+    - blas =*.*=mkl
     - click >=7.0
     - cmocean >=2.0
     - dask >=1.2
     - fiona >=1.8
-    - gdal >=2.4,<3.0
-    - matplotlib-base >=3.0
+    - gdal >=2.4
+    - jdcal >=1.4.1
+    - geopandas >=0.6.2
+    - jsonschema >=3.2.0
+    - matplotlib >=3.0
```

Comment:
Does matplotlib-base not work? matplotlib drags in qt

Reply:
In that case, should we switch to matplotlib-base in environment.yml?

Reply:
Yes, please!

Reply:
Changed (back) to `matplotlib-base` here, and opened https://github.com/dcs4cop/xcube/issues/361 for corresponding change in `environment.yml`